migrations/20170802152927-add_imc_profile_migration.js

```diff
@@ -1,27 +1,77 @@
 module.exports = {
   up: function (qi, Sequelize) {
-    return qi.addColumn('Profiles', 'botAccess', {
-      type: Sequelize.ENUM('r', 'rw'),
-      defaultValue: 'r',
+    qi.sequelize.query('SELECT * FROM "Profiles" LIMIT 1',
```

Comment:
Better to use qi.describeTable than to try to select a record from the table. See /refocus/migrations/20161115215050-fixglobalconfig.js for an example where that is used to check for existence of column.